packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -165,6 +170,7 @@ export function useSpecificEditorSettings() {
 			// I wonder if they should be set in the post editor too
 			__experimentalArchiveTitleTypeLabel: archiveLabels.archiveTypeLabel,
 			__experimentalArchiveTitleNameLabel: archiveLabels.archiveNameLabel,
+			__experimentalArchiveDescription: archiveLabels.archiveDescription,
```

Comment:
Let's avoid adding random block editor settings like that that are only needed for a given block.

I'm not sure I understand the PR probably but from my reading, you want to retrieve the "template" description kind of and use it in a block.

I've just merged https://github.com/WordPress/gutenberg/pull/63478 which fixes this for the other two labels and the QueryTitle block and I believe it might be possible to reuse the same hook (that is now in block-library) directly in the post author and term description blocks here instead of adding a new setting. We could add a new returned value to the hook.